Source/WebCore/rendering/RenderLayer.h

```diff
@@ -120,12 +120,31 @@ enum ShouldApplyRootOffsetToFragments {
     IgnoreRootOffsetForFragments
 };
 
-enum class RequestState {
-    Unknown,
-    DontCare,
-    False,
-    True,
-    Undetermined
+struct RequestState {
+    enum State {
```

Comment:
Going from `enum class` to old style `enum` seems like a step in the wrong direction?

Reply:
This change was removed.